### PR TITLE
[PowerToys] Add Command Not Found docs

### PR DESCRIPTION
**Pull request by stefansjfw**
To be merged only with the PowerToys 0.77 release

### CODE REVIEW

File: hub/powertoys/cmd-not-found.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+# Command Not Found utility
+
+A PowerShell 7 module that detects an error thrown by a command and suggests a relevant WinGet package to install, if available.
+
+![AlwaysOnTop screenshot.](../images/pt-cmd-not-found.png)
+
+## Requirements
+ - [PowerShell 7](https://learn.microsoft.com/en-us/PowerShell/scripting/install/installing-PowerShell-on-windows?view=PowerShell-7.4)
```
Comment (alvinashcraft) [inline hub/powertoys/cmd-not-found.md]:
```suggestion
 - [PowerShell 7](/PowerShell/scripting/install/installing-PowerShell-on-windows?view=PowerShell-7.4)
```

Is the `?view=PowerShell-7.4` query param needed, or can we let it default to the latest PowerShell docs?

File: hub/powertoys/cmd-not-found.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ - PSCommandNotFoundSuggestion
+
+After that, PowerShell profile file will be appended with following block of PowerShell commands:
+```
+#34de4b3d-13a8-4540-b76d-b9e8d3851756 PowerToys CommandNotFound module
+Import-Module "<powertoys install dir>/WinGetCommandNotFound.psd1"
+#34de4b3d-13a8-4540-b76d-b9e8d3851756
+```
+
+Note: If there is no profile file at the time of installing the module, profile file will be created. After installing the module, restart PowerShell session to use the module.
```
Comment (alvinashcraft) [inline hub/powertoys/cmd-not-found.md]:
```suggestion
**Note:** If there is no profile file at the time of installing the module, profile file will be created. After installing the module, restart PowerShell session to use the module.
```
Is bold sufficient, or do you think it should be formatted as a `[!NOTE]`?

File: hub/powertoys/cmd-not-found.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+![AlwaysOnTop screenshot.](../images/pt-cmd-not-found.png)
+
+## Requirements
+ - [PowerShell 7](https://learn.microsoft.com/en-us/PowerShell/scripting/install/installing-PowerShell-on-windows?view=PowerShell-7.4)
+ - [PowerShell Microsoft.WinGet.Client module](https://www.powershellgallery.com/packages/Microsoft.WinGet.Client)
+
+## Install the module
+
+To install the Command Not Found module, navigate to PowerToys settings Command Not Found page and press Install button. On module install, PowerShell 7 experimental features needed for the module to function will be enabled:
```
Comment (alvinashcraft) [inline hub/powertoys/cmd-not-found.md]:
```suggestion
To install the Command Not Found module, navigate to the Command Not Found page in PowerToys settings and select the **Install** button. Once the module installation has completed, PowerShell 7 experimental features needed for the module to function will be enabled:

```

Comment (Jay-o-Way) [inline hub/powertoys/cmd-not-found.md]:
```suggestion
To install the Command Not Found module, go to PowerToys settings Command Not Found page and select **Install**. After installation, PowerShell 7 experimental features needed for the module to function will be enabled:
```

File: hub/powertoys/cmd-not-found.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+#34de4b3d-13a8-4540-b76d-b9e8d3851756 PowerToys CommandNotFound module
+Import-Module "<powertoys install dir>/WinGetCommandNotFound.psd1"
+#34de4b3d-13a8-4540-b76d-b9e8d3851756
+```
+
+Note: If there is no profile file at the time of installing the module, profile file will be created. After installing the module, restart PowerShell session to use the module.
+
+## Uninstall the module
+
+To uninstall the Command Not Found module, navigate to PowerToys settings Command Not Found page and press Uninstall button. On module uninstall, previously added block of commands will be removed from PowerShell profile file.
```
Comment (alvinashcraft) [inline hub/powertoys/cmd-not-found.md]:
```suggestion
To uninstall the Command Not Found module, navigate to the Command Not Found page in PowerToys settings and select the **Uninstall** button. Once the module uninstall has completed, the block of commands previously added will be removed from the PowerShell profile file. 
```

**Review by alvinashcraft — CHANGES_REQUESTED** [review]

File: hub/powertoys/cmd-not-found.md (hunk 4 above)
Comment (Jay-o-Way) [inline hub/powertoys/cmd-not-found.md]:
```suggestion
To uninstall the Command Not Found module, go to PowerToys settings Command Not Found page and select **Uninstall**. After uninstallation, previously added block of commands will be removed from PowerShell profile file. 
```

File: hub/powertoys/cmd-not-found.md (hunk 3 above)
Comment (alvinashcraft) [reply hub/powertoys/cmd-not-found.md]: I just left a suggestion for this same block. Can you combine the suggestions into an updated paragraph, @stefansjfw?

File: hub/powertoys/cmd-not-found.md (hunk 2 above)
Comment (Jay-o-Way) [inline hub/powertoys/cmd-not-found.md]:
```suggestion
Note: The profile file will be created if needed. Restart PowerShell session to use the module.
```

File: hub/powertoys/cmd-not-found.md (hunk 1 above)
Comment (Jay-o-Way) [inline hub/powertoys/cmd-not-found.md:16]:
```suggestion
 - [PowerShell 7](https://learn.microsoft.com/PowerShell/scripting/install/installing-PowerShell-on-windows?view=PowerShell-7.4)
```

File: hub/powertoys/cmd-not-found.md (hunk 3 above)
Comment (Jay-o-Way) [reply hub/powertoys/cmd-not-found.md]: Haha, we're thinking the same thing! :)

File: hub/powertoys/cmd-not-found.md (hunk 2 above)
Comment (Jay-o-Way) [reply hub/powertoys/cmd-not-found.md]: Honestly and personally, this actually feels more like a normal paragraph to me.

Comment (stefansjfw) [reply hub/powertoys/cmd-not-found.md]: Bold is ok. I'd go with bold to outline this, especially "restart PowerShell" part

**Comment by stefansjfw** [comment]
I believe I addressed all the comments. Thanks for the suggestions!

**Review by alvinashcraft — APPROVED** [review]
This looks good to me. We'll see if @Jay-o-Way has any other suggestions. Otherwise, I'll merge when the release goes out.

**Comment by Jay-o-Way** [comment]
Looks good to me :)

File: hub/powertoys/cmd-not-found.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+title: PowerToys Command Not Found for Windows
+description: A PowerShell 7 module that detects an error thrown by a command and suggests a relevant WinGet package to install, if available.
+ms.date: 02/01/2024
+ms.topic: article
+no-loc: [PowerToys, Windows, Command Not Found, Win]
+---
+
+# Command Not Found utility
+
+A PowerShell 7 module that detects an error thrown by a command and suggests a relevant WinGet package to install, if available.
```
Comment (NathanD-GDS) [inline hub/powertoys/cmd-not-found.md]:
```suggestion
A PowerShell 7 module that detects command-line errors and suggests a relevant WinGet package to install, if available.
```

This is just a small suggestion but I'm not sure about 'thrown' as a verb to describe errors. I know it's used for [exceptions in these docs](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/exception-throwing) but I can't see it in many other Microsoft docs.

File: hub/powertoys/cmd-not-found.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+## Install the module
+
+To install the Command Not Found module, navigate to the Command Not Found page in PowerToys settings and select the **Install** button. Once the module installation has completed, PowerShell 7 experimental features needed for the module to function will be enabled:
+
+ - PSFeedbackProvider
+ - PSCommandNotFoundSuggestion
+
+After that, PowerShell profile file will be appended with following block of PowerShell commands:
+```
```
Comment (Jay-o-Way) [inline hub/powertoys/cmd-not-found.md]: What's the language for this code snippet? psh?

File: hub/powertoys/cmd-not-found.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+no-loc: [PowerToys, Windows, Command Not Found, Win]
+---
+
+# Command Not Found utility
+
+A PowerShell 7 module that detects an error thrown by a command and suggests a relevant WinGet package to install, if available.
+
+![Command Not Found screenshot.](../images/pt-cmd-not-found.png)
+
+## Requirements
```
Comment (Jay-o-Way) [inline hub/powertoys/cmd-not-found.md:18]:
```suggestion
## Requirements

```

**Review by Jay-o-Way — COMMENTED** [review]
Nit bits

File: hub/powertoys/cmd-not-found.md (hunk 6 above)
Comment (stefansjfw) [reply hub/powertoys/cmd-not-found.md]: yes

Comment (Jay-o-Way) [reply hub/powertoys/cmd-not-found.md]:
```suggestion

```psh
```

**Pull request merged**